### PR TITLE
Guus

### PR DESCRIPTION
**Pull request by GuusAvis**
Open question is still where the raw data should go. It's 1.6 GB so uploading it here may be not so suitable.

Don't merge yet; still need to test the code (netsquid-treecode is not yet deployed so cannot test the install script) and there is still a #fixme in the readme for the reference to the raw data.

### CODE REVIEW

**Comment by GuusAvis** [comment]
- netsquid-treecode is now deployed and therefore the code should be successfully executable now (though one should put the `number_of_states` argument much smaller if one just wants to test the python script).
- I have put the DOI that has been reserved for the data set in the readme, although it currently doesn't lead anywhere yet as the data is not yet online.

**Comment by bernwo** [comment]
@GuusAvis I read the `README.md` and I think it looks good. I also moved things around a little bit, i.e., move all the stuff into a separate folder so that the repository is cleaner because eventually I need to add my code too, which I will put in a separate folder.

Is the code now tested and running on your Linux machine? If so, then I shall approve and merge this PR.

File: README.md
Comment (bernwo) [inline README.md:1]: Looks good to me

File: reencoding_error_probability/processed_data_for_paper_reorganized.odt
Comment (bernwo) [inline reencoding_error_probability/processed_data_for_paper_reorganized.odt:1]: I had to update the filename to be consistent with the `README.md`. I assume the original file extension `.ods` was a typo and that it's supposed to be `.odt`.

Comment (GuusAvis) [reply reencoding_error_probability/processed_data_for_paper_reorganized.odt:1]: No it's actually supposed to be .ods (it's an OpenDocument Spreadsheet, not OpenDocument Text).

**Comment by GuusAvis** [comment]
Yes the code has been tested now!

File: reencoding_error_probability/processed_data_for_paper_reorganized.odt
Comment (bernwo) [reply reencoding_error_probability/processed_data_for_paper_reorganized.odt:1]: Alright, addressed this in [e21ea1d](https://github.com/bernwo/code-concatenated-quantum-repeater/pull/1/commits/e21ea1d0251de8ee6e56b98572f895f66a14f3cb)

**Review by bernwo — APPROVED** [review]

**Pull request merged**